### DISCUSSION TITLE
Releases not signed by Google Play

### DISCUSSION THREAD

**Discussion by jeuxdaily** [airsdk/Adobe-Runtime-Support#2381] [General]
hi

I have some apps in my google console

So when I want to update, I get the following problem 

this problem for all apps

My Apps was creating by Adobe Animate cc

Releases not signed by Google Play

Google isn't protecting the app signing key for your app. Opt in to use Android App Bundles.

thanks

**Comment by FliplineStudios** [comment]
Just a quick note that for existing apps that were originally APKs, Google lets you continue to just submit APKs like before instead of switching to AAB, we've recently done that for their new required nearly-annual updates.  

For Android App Bundles and Play App Signing, there's a tutorial on airsdk.dev about packaging in general here:
https://airsdk.dev/docs/tutorials/platform/android/packaging-android-app-bundles

There should be info in there explaining how to package as AAB and how to start using Play App Signing under the "Convert Existing Certificates" section.  I'm not sure if there are any extra steps needed for updating older APK releases that previously didn't use Play App Signing though (maybe someone can chime in if there's anything in particular to look out for with that?)

**Reply by jeuxdaily** [reply]
thanks 
Enter password for store 'filecertf.p12' this password for certificate file 
Enter password for key = what password should be entered

thanks

**Comment by jeuxdaily** [comment]
Thank you very much and I wish you success
